doc/API.md

```diff
@@ -1,161 +1,8 @@
-
-### Usage example
-
-```js
-  // initialize a torque layer that uses the CartoDB account details and SQL API to pull in data
-  var torqueLayer = new L.TorqueLayer({
-    user       : 'viz2',
-    table      : 'ow',
-    cartocss:  CARTOCSS
-  });
-```
-
-### Options
-
-##### Provider options
-
-| Option    | type       | Default   | Description                            | Options |
-|-----------|:-----------|:----------|:---------------------------------------|---------|
-| provider  | string     | ```sql_api```   | Where is the data coming from    | `sql_api`, `url_template`, or `windshaft` |
-
-##### CartoDB data options (SQL API provider)
-
-| Option    | type       | Default   | Description                            |
-|-----------|:-----------|:----------|:---------------------------------------|
-| user_name      | string     | ```null```      | CartoDB account name. Found from: http://accountname.cartodb.com|
-| table_name     | string     | ```null```      | CartoDB table name where data is found  |
-| query       | string     | ```null```      | SQL query to be performed to fetch the data. You must use this param or table, not at the same time |
-| cartocss  | string     | ```null```      | CartoCSS style for this map |
-| loop      | boolean    | ```true```      | If ```false```, the animation is paused when it reaches the last frame |
-
-
-### Time methods
-
-| Method    | options    | returns   | Description                            |
-|-----------|:-----------|:----------|:---------------------------------------|
-| ```setStep(step)``` | ```time numeric```    | ```this```   | sets the animation to the step indicated by ```step```, must be between 0 and ```steps```|
-| ```play()```| | ```this```| starts the animation
-| ```stop()```| | ```this```| stops the animation and set time to step 0
-| ```pause()```| | ```this```| stops the animation but keep the current time (play enables the animation again)
-| ```toggle()```| | ```this```| toggles (pause/play) the animation 
-| ```getStep()``` | | current animation step (integer)   | gets the current animation step
-| ```getTime()``` | | current animation time (Date) | gets the real animation time
-| ```isRunning()``` |  | `true`/`false` | describes whether the Torque layer is playing or is stopped 
-
-### Layer control methods
-
-| Method    | options    | returns   | Description                            |
-|-----------|:-----------|:----------|:---------------------------------------|
-| ```hide()``` | none    |```this```  | hides the Torque layer |
-| ```show()```| none| ```this``` | shows the Torque layer |
-
-### Style methods 
-
-| Method    | options    | returns   | Description                            |
-|-----------|:-----------|:----------|:---------------------------------------|
-| ```setCartoCSS(cartocss)``` | ```cartocss string```    | ```this```   | style the map rendering using client-side cartocss (not available with named maps) | 
```

Comment:
Added link to Named map section http://docs.cartodb.com/cartodb-platform/maps-api/named-maps/